integrations/tensorflow/compiler/dialect/tf_strings/ir/ops.td

```diff
@@ -44,6 +44,36 @@ def TFStrings_ToStringOp : TFStrings_Op<"to_string"> {
     OpBuilderDAG<(ins "Value":$value)>];
 }
 
+def TFStrings_GatherOp : Op<TFStrings_Dialect, "gather"> {
+  let summary = [{gathers all the strings from a Tensor by index}];
+  let description = [{
+    Gathers all the strings from a Tensor by ID.
+  }];
+
+  let arguments = (ins TFStrings_StringTensor:$dict,
+                  TFStrings_ValueTensor:$ids);
+
+  let results = (outs
+    TFStrings_StringTensor:$result
+  );
+}
+
+def TFStrings_ConcatOp : Op<TFStrings_Dialect, "concat"> {
+  let summary = [{
+  concatenates the strings in the tensor along the last dimension
+  }];
```

Comment:
formatting

```suggestion
  let summary = [{
    Concatenates the strings in the tensor along the last dimension.
  }];
```

OR shorten and use quotes so the summary isn't just the same as the description?